tutorials/guide-to-breaches.md

```diff
@@ -14,18 +14,47 @@ Ships on the Ames network sometimes need to reset their continuity. A personal b
 
 Personal breaches often fix connectivity issues, but should only be used as a last resort. Before performing a personal breach, look at alternative fixes in the [Ship Troubleshooting](../ship-troubleshooting) guide. Also reach out for help in `~/~dopzod/urbit-help`, or, failing that, in the `#ship-starting-support` channel in our [Discord server](https://discord.gg/n9xhMdz) to see if there is another option.
 
-To perform a personal breach, follow the steps below.
+There are two types of personal breaches - one where your Ethereum ownership address
+remains the same, and one where you are switching to a new Ethereum ownership
+address. We make the emphasis about the Ethereum _ownership_ address as
+changing your [proxies](@/glossary/proxies.md) does not require a breach.
+
+If you will be keeping the same Ethereum ownership address and would like to perform a
+personal breach, follow the steps below.
 
 - Go to [bridge.urbit.org](https://bridge.urbit.org) and log into your identity.
 - Click on `OS: Urbit OS Settings` at the bottom, then click `Reset Networking Keys`.
 - Check the `Breach Continuity` box. Click `Reset Networking Keys`, and then click `Send Transaction` and wait for the progress bar to appear.
-- Click `Download Arvo Keyfile`.
-- Delete or archive your old pier.
-- Create a new pier by booting your ship with your new keyfile.
+- Download your new keyfile following these instructions: [Generate your
+  keyfile](@/using/operations/using-bridge.md#generate-your-keyfile).
+- Delete or archive your old [pier](@/glossary/pier).
+- Proceed to [boot your ship](@/using/install.md#boot-your-planet) with the new keyfile.
+- Delete your keyfile after successfully booting.
 - Rejoin your favorite chat channels and subscriptions.
 
-There are two types of personal breaches: changing private keys, and changing
-the Ethereum address that holds the Urbit ID.
+If you are switching to a new Ethereum ownerhsip address, you will necessarily be
+performing a personal breach as well - there is no way to change the ownership
+address without also breaching. The process here is slightly different.
```

Comment:
You'll want to revise this based on urbit/bridge#525.

Reply:
Awesome. I'll wait til this PR is merged before making the edits in case the wording on Bridge changes in the meantime.